.github/workflows/ci.yml

```diff
@@ -602,58 +540,14 @@ jobs:
       docker-cache: ${{ needs.build-info.outputs.docker-cache }}
       disable-airflow-repo-cache: ${{ needs.build-info.outputs.disable-airflow-repo-cache }}
 
-  wait-for-prod-images:
```

Comment:
Same here. All PROD images are now built in the same workflow and we replace waiting with regular "depends-on".